BPRBlazor/Models/ArchitecturalModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace BPRBlazor.Models;
+
+public class ArchitecturalModel
+{
+    [Required(ErrorMessage = "Please give the model a name")]
+    public string Name { get; set; } = string.Empty;
+
+    [Required]
```

Comment:
Isn't required useless here as you create it as new()?

Reply:
I suppose it is in a way, but without it there'd be nothing stopping you from assigning it to null.

Reply:
I guess just leave it be for now